build.gradle

```diff
@@ -75,6 +75,7 @@ dependencies {
 
     provided "junit:junit:4.12", "org.hamcrest:hamcrest-core:1.3"
     compile "org.objenesis:objenesis:2.4"
+    compile 'net.jodah:typetools:0.4.8'
```

Comment:
Is this library really required, would like to not add a library for 1 method invocation.